### PR TITLE
PanelEdit: Fixes resize pane border and spacing issues  

### PR DESCRIPTION
**Pull request by torkelo**
https://github.com/grafana/grafana/pull/54420 broke 3 things in panel edit (2 fixed so far in this PR) it also duplicated the resizer styling and the callbacks so that it's now something we have to maintain in two places (SplitView and SplitViewWrapper), that does not feel ideal. 

* [x] Fixes border bug
* [x] Fixes a spacing bug (left split pane had overflow overflowY added which caused space for a scrollbar which caused unaligned spacing for the visualzation view in pane ledit)
* [ ] Changing size of split pane is not saved like before, reloading panel edit rests it to the default. In 9.1 it still works and it remembers the split size 

Below you see the border (should not be there), and the left side space is not equal to right side (should be 16px spacing between panes but it's 24)
image

### CODE REVIEW

File: public/app/core/components/SplitPaneWrapper/SplitPaneWrapper.tsx [hunk 1]
```diff
@@ -161,25 +161,6 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
...
-          left: 50%;
-          transform: translateX(-50%);
-        }
-
-        &::after {
-          height: 200px;
-          width: 4px;
-        }
-      `
-    ),
```
Comment (torkelo) [inline public/app/core/components/SplitPaneWrapper/SplitPaneWrapper.tsx:182]: I think adding this back and removing SplitView and figuring out how to make SplitPaneWrapper more generic might be one path forward? or creating a standalone VerticalSplitView for explore

File: public/app/core/components/SplitPaneWrapper/SplitView.tsx [hunk 2]
```diff
@@ -83,7 +83,6 @@ export const SplitView = ({ uiState: { rightPaneSize }, children, minSize = 200,
       minSize={minSize}
       maxSize={width - minSize}
       resizerClassName={useStyles2(getResizerStyles(hasSplit))}
-      paneStyle={{ overflow: 'auto', display: 'flex', flexDirection: 'column', overflowY: 'scroll' }}
```
[the remark this answers is not in the document]
Comment (gelicia) [reply public/app/core/components/SplitPaneWrapper/SplitView.tsx:86]: Created https://github.com/grafana/grafana/issues/56213 to work on this

[the remark this answers is not in the document]
Comment (torkelo) [reply public/app/core/components/SplitPaneWrapper/SplitView.tsx:86]: @gelicia please help in this PR, needs to be fixed asap so we can backport this in 9.2 stable to we don't ship 9.2 with these bugs in panel edit and explore

[the remark this answers is not in the document]
Comment (gelicia) [reply public/app/core/components/SplitPaneWrapper/SplitView.tsx:86]: @torkelo The team discussed this and decided the best bet is to revert the split resize PRs so we can work on this outside of the release. We will be doing that shortly.

**Pull request closed without merging**